int16 cliff1
int16 cliff2
int16 cliff3
int16 cliff4
